packages/mssql/configuration-schema.json

```diff
@@ -1,14 +1,27 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$schema": "https://json-schema.org/draft-07/schema#",
   "properties": {
     "server": {
       "title": "Server URL",
-      "type": "string",
       "description": "The database instance server URL or IP address",
-      "format": "uri",
+      "oneOf": [
```

Comment:
@elias-ba i am curious if this change will break lightning credential form ?

Reply:
We're going to simplify this so it doesn't matter now :)